Dockerfile.konflux

```diff
@@ -0,0 +1,9 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.21 AS builder
+WORKDIR /go/src/github.com/stolostron/submariner-addon
+COPY . .
+ENV GO_PACKAGE github.com/stolostron/submariner-addon
+RUN make GO_BUILD_FLAGS=-mod=mod build --warn-undefined-variables
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
Shouldn't we set this to els image, or is that not ready yet?

Reply:
One thing at a time :wink: https://github.com/stolostron/submariner-addon/pull/1404#discussion_r1634337512